aisikl/app.py

```diff
@@ -628,7 +628,7 @@ def shell_exec(self, content_type, file_name, new_window_name, width,
         if content_type == 'webui/execCustomFunction':
             raise AISParseError("webui/execCustomFunction is not supported")
 
-        if width or height:
+        if (width or height) and not (width == -1 and height == -1):
```

Comment:
Tu vobec netusim preco hadzeme Error ked AIS vrati `width` alebo `height` no kedze vracal `-1` co mi pride ako `None` v nejakom C-cku tak som to oIFoval, dufam, ze to nic nepokazilo.